Library/Formula/ssldump.rb

```diff
@@ -1,9 +1,9 @@
-require 'formula'
-
 class Ssldump < Formula
-  homepage 'http://www.rtfm.com/ssldump/'
-  url 'http://www.rtfm.com/ssldump/ssldump-0.9b3.tar.gz'
-  sha1 'a633a9a811a138eac5ed440d583473b644135ef5'
+  homepage "http://www.rtfm.com/ssldump/"
+  url "http://www.rtfm.com/ssldump/ssldump-0.9b3.tar.gz"
+  sha1 "a633a9a811a138eac5ed440d583473b644135ef5"
+
+  depends_on "openssl"
 
   # reorder include files
   # http://sourceforge.net/tracker/index.php?func=detail&aid=1622854&group_id=68993&atid=523055
```

Comment:
Also would be good to submit an upstream patch for the `./configure` `inreplace` below too.